lld/ELF/LinkerScript.cpp

```diff
@@ -801,7 +801,7 @@ static OutputDesc *addInputSec(StringMap<TinyPtrVector<OutputSection *>> &map,
       auto *firstIsec = cast<InputSectionBase>(
           cast<InputSectionDescription>(sec->commands[0])->sectionBases[0]);
       OutputSection *firstIsecOut =
-          firstIsec->flags & SHF_LINK_ORDER
```

Comment:
Why is random change made to lld/ELF?

`[-+*%] \w+ \?'` is extensively used and I don't see we add `)`

Reply:
Hi @MaskRay ,

the PR modifies all the files with non-trivial ternary operators given by Cppcheck ([issue](https://github.com/llvm/llvm-project/issues/85868)).
You can find the Cppcheck report inside the comments thread of the issue.

Reply:
A CppCheck issue may or may not be useful for the project. Projects have different code styles. Applying a random static analyzer report may not be useful.